app/controllers/ApplicationController.java

```diff
@@ -23,16 +19,12 @@
 public final class ApplicationController extends SunriseController {
 
     @Inject
-    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree, final Configuration configuration) {
-        super(client, categoryTree, configuration);
+    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree,
```

Comment:
In nightfall I had just a class ControllerDeps and never worried about it. Inside the super class it was unpacking this and provides methods to access it.